### ISSUE TITLE
confuse

### ISSUE THREAD

**Issue by wymxz** [facebookresearch/adaptive-span#23]
Are the results of dev and test on the test data set?

Experiment | #params | dev | test
enwik8 | 38M | 1.04 bpb | 1.02 bpb
enwik8_large | 209M | 1.00 bpb | 0.98 bpb
text8 | 39M | 1.05 bpc | 1.11 bpc
text8_large | 209M | 1.01 bpc | 1.07 bpc

 Do they need to be evaluated multiple times on the test? When I reproduce the model's train and valid bpcs are much larger than those obtained on the test, is it normal?

**Comment by tesatory** [comment]
Here "dev" means the validation set, and "test" is the test set. To obtain the test bpc, you have to run the evaluation code like here https://github.com/facebookresearch/adaptive-span/blob/master/experiments/enwik8.sh#L41
